README.rst

```diff
@@ -101,14 +101,15 @@ Tracing an RPC
 If you want to make an RPC and continue an existing trace, you can inject the current span into the RPC. For example, if making an http request, the following code will continue your trace across the wire:
 
 .. code-block:: python
+    from future.utils import listitems # for python3 compatibility
 
     @tracer.trace()
     def some_view_func(request):
         new_request = some_http_request
         current_span = tracer.get_span(request)
         text_carrier = {}
         opentracing_tracer.inject(span, opentracing.Format.TEXT_MAP, text_carrier)
-        for k, v in text_carrier.iteritems():
+        for k, v in listitems(text_carrier):
```

Comment:
Shouldn't we preserve the existing behaviour of not converting to a list?

e.g.
```
from future.utils import iteritems
...
for k, v in iteritems(text_carrier):
```